cmd/cli/main.go

```diff
@@ -75,11 +75,11 @@ func DeployAction(c *cli.Context) error {
 func UpdateAction(c *cli.Context) error {
 	update := UpdateJob{}
 
-	jobName := c.String("jobname")
-	if len(jobName) == 0 {
-		return cli.NewExitError("unspecified flag 'jobname'", -1)
+	jobNameBase := c.String("jobname-base")
```

Comment:
Maybe `job-name-base` is a better param?

Reply:
At the moment in other places we also use jobname appended, like query command. Also some are camel and some are snake case. Shall I move all to Camel case (e.g. jobNameBase)?

Reply:
We're still talking about the cmd line arguments right? I don't think that we should use camel case for the cmd line arguments. Dash separated seems to be the more widely used approach. using camelcase for variables itself is fine though.

Reply:
Alright, then I'll change all to dash separated (some lingering with camelcase atm)